linux/ibus-keyman/tests/scripts/run-single-test.sh

```diff
@@ -34,6 +34,8 @@ function help() {
   echo "  --wayland               run tests with Wayland"
   echo "  --x11                   run tests with X11"
   echo "  --env <envfile>         Name of the file containing environment variables to use"
+  echo "  --check <pidfile>       Name of the file containing pids to check are running"
+  echo "  --cleanup <file>        Name of the file containing cleanup of processes"
```

Comment:
Does the usage on l.24 get longer with these additional args? :)

Reply:
Thanks!